app/models/match_decisions/record_client_housed_date_housing_subsidy_administrator.rb

```diff
@@ -106,20 +106,20 @@ def holds_voucher?
 
     class StatusCallbacks < StatusCallbacks
       def pending
-        match.client.update(holds_voucher_on: Date.current) unless match.has_buildings?
+        match.client.update(holds_voucher_on: Date.current, holds_internal_cas_voucher: true) unless match.has_buildings?
       end
 
       def other_clients_canceled
         match.cancel_opportunity_related_matches!
       end
 
       def completed
-        match.client.update(holds_voucher_on: nil)
+        match.client.update(holds_voucher_on: nil, holds_internal_cas_voucher: nil)
```

Comment:
I think there is a possible problem with clearing `holds_voucher_on` in the step... I don't think the sync from project_client will happen to set it back?

Reply:
Can you explain a bit more?  This should remove the "holds voucher" when proceeding to the next step, if the non-HMIS client has a voucher, I believe it will be re-added on next sync.